legend-engine-xt-relationalStore-executionPlan-connection/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/connection/authentication/strategy/ApiTokenAuthenticationStrategy.java

```diff
@@ -61,7 +61,6 @@ public Pair<String, Properties> handleConnection(String url, Properties properti
     {
         ApiTokenCredential apiTokenCredential = this.resolveCredential(properties, this.apiToken);
         Properties connectionProperties = new Properties();
-        connectionProperties.putAll(properties);
```

Comment:
SQL endpoints do not support extra JDBC properties. Only passing properties we need (i.e. to token)